semantics/language/common/expr/reference.k

```diff
@@ -67,6 +67,7 @@ module C-COMMON-EXPR-REFERENCE
      }*/
      rule *(tv(Loc:SymLoc, T:PointerType))
           => checkRestrict(Loc)
+          ~> checkLoc(Loc)
```

Comment:
Sure this is necessary? Bad addresses should in theory be caught well before this point.